main.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, jsonify, request
+from numpy import sqrt, square
+app = Flask(__name__)
+
+
+@app.route("/name", methods=["GET"])
+def getName():
+    data = {
+        "name": "Michelle"
+    }
+    return jsonify(data)
+
+
+@app.route("/hello/<name>", methods=["GET"])
+def getHello(name):
+    data = {
+        "message": "Hello there, {0}".format(name)
+    }
+    return jsonify(data)
+
+
+@app.route("/distance", methods=["POST"])
+def postDistance():
+    r = request.get_json()
+    d = sqrt(square((r["a"][0]-r["b"][0]))+square((r["a"][1]-r["b"][1])))
```

Comment:
you should include spaces around math operators, for ex `a + b` as opposed to `a+b`